web/src/components/modal-event-details/index.tsx

```diff
@@ -8,7 +8,11 @@ import { InfoIcon } from '@components/icons/InfoIcon';
 import { RepeatEvent } from '@constants';
 import { yupResolver } from '@hookform/resolvers/yup';
 import { IModalEventDetails } from '@interfaces';
-import { TextField, Tooltip } from '@mui/material';
+import {
+  Autocomplete as MuiAutocomplete,
```

Comment:
Any reason why we're renaming this component as `MuiAutocomplete`?

Reply:
yes, there is already an Autocomplete:
`import Autocomplete from 'react-google-autocomplete';`

Reply:
Oh yeah! Totally forgot about that one hehe